week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,20 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+function percentage(mentorCount, studentsCount) {
+  var mentorPercent = Math.round(
+    mentorCount / (mentorCount + studentsCount) * 100
+  );
+  var studentsPercent = Math.round(
+    studentsCount / (mentorCount + studentsCount) * 100
+  );
+  /*console.log(
+    "mentorPercent = " + mentorPercent,
+    "studentsPercent = " + studentsPercent
+  );*/
+  printMessage(mentorPercent, studentsPercent);
```

Comment:
Whenever possible, try to ensure that your functions return a value and do not perform side effects. 

In this example it would be better if the `percentage` function returns a percentage value. You could then get the value, assign it to a variable, and use `console.log` to print it.